tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/ASM/AST/SyntaxKindProxy.cs

```diff
@@ -0,0 +1,62 @@
+// <copyright file="SyntaxKindProxy.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate.ASM.AST
```

Comment:
NIT: use scoped namespaces in this and other files of the PR.

Reply:
Done in 8e67bc2d3bf40bedb468c49a28e474862c23c105